Marlin/Configuration.h

```diff
@@ -577,11 +577,11 @@
  *
  * A4988 is assumed for unspecified drivers.
  *
- * Options: A4988, A5984, DRV8825, LV8729, L6470, TB6560, TB6600, TMC2100,
+ * Options: A4988, A5984, DRV8825, LV8729, ST_L6470, TB6560, TB6600, TMC2100,
```

Comment:
This name change really isn't needed. There shouldn't be any naming conflict with the Arduino-L6470 library with the proposed change to `drivers.h`. I assume you reverted the name change so that you could bring in your additional changes, and not because it was a problem…?